e2e-tests/tests/telemetry-transfer/01-assert.yaml

```diff
@@ -15,65 +15,10 @@ spec:
     singular: perconapgcluster
   scope: Namespaced
 ---
-kind: Deployment
-apiVersion: apps/v1
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
 metadata:
-  name: percona-postgresql-operator
-  generation: 1
-  labels:
-    helm.sh/chart: fake-operator-chart
-    app.kubernetes.io/component: operator
-    app.kubernetes.io/instance: percona-postgresql-operator
-    app.kubernetes.io/name: percona-postgresql-operator
-    app.kubernetes.io/part-of: percona-postgresql-operator
-    pg.percona.com/control-plane: postgres-operator
-  annotations:
-    deployment.kubernetes.io/revision: '1'
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app.kubernetes.io/component: operator
-      app.kubernetes.io/instance: percona-postgresql-operator
-      app.kubernetes.io/name: percona-postgresql-operator
-      app.kubernetes.io/part-of: percona-postgresql-operator
-      pg.percona.com/control-plane: postgres-operator
-  template:
-    metadata:
-      creationTimestamp: null
-      labels:
-        app.kubernetes.io/component: operator
-        app.kubernetes.io/instance: percona-postgresql-operator
-        app.kubernetes.io/name: percona-postgresql-operator
-        app.kubernetes.io/part-of: percona-postgresql-operator
-        pg.percona.com/control-plane: postgres-operator
-    spec:
-      containers:
-        - name: operator
-          ports:
-            - name: metrics
-              containerPort: 8080
-              protocol: TCP
-          env:
-            - name: PGO_TARGET_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  apiVersion: v1
-                  fieldPath: metadata.namespace
-            - name: PGO_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  apiVersion: v1
-                  fieldPath: metadata.namespace
-            - name: CRUNCHY_DEBUG
-              value: 'true'
-            - name: DISABLE_TELEMETRY
-              value: 'false'
-            - name: PERCONA_VS_FALLBACK_URI
```

Comment:
So 2 possible ways: 
1. We somehow parameterize operator namespace in assert (I didn't manage to use variable, so it looks like I need to patch file directly)
2. Add 2 separate steps: 
1 - run operator. 
2. - update deploy operator with PERCONA_VS_FALLBACK_URI
In second case we still  will use script to check particular values, but we can check separately operator deploy status and PERCONA_VS_FALLBACK_URI value. In addition we can avoid extra if for telemetry in case of CW, because we can use `kubectl set env ` instead of using unpredictable insert (it will be broken if we add new env in operator file)